package.json

```diff
@@ -21,7 +21,7 @@
     "dependencies": {
         "@types/uuid": "^8.3.4",
         "cosmian_cover_crypt": "^6.0.8",
-        "cosmian_findex": "^0.7.2",
+        "cosmian_findex": "file:../findex/pkg/bundler",
```

Comment:
Need to change this after the merge of the MR in Findex

Reply:
yes